addon/test/test-share-prompt.js

```diff
@@ -0,0 +1,168 @@
+/*
```

Comment:
:+1: and tests pass with `jpm test --verbose --binary /Applications/FirefoxDeveloperEdition.app/Contents/MacOS/firefox-bin`